src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/app_start/routing/web/WebUserGroupRouteConfig.kt

```diff
@@ -23,6 +23,11 @@ object WebUserGroupRouteConfig : RouteConfig
                     UserGroupController::class, "associatePermission",
                     method = HttpMethod.post)
                     .json()
+                    .secure(usersManage),
+            WebEndpoint("/typeahead/roles/",
+                    UserGroupController::class, "getAllRoleNames")
```

Comment:
So, is there a proper boundary point between 'user groups' (our internal term) and 'roles'  (what's presented to users). Or are we going to use 'roles' at all levels to designate a user group which is not an identity group? Latter makes sense I guess. But in that case should the /user-groups/report-readers/ endpoint change to /roles/report-readers/ ?

Reply:
I think this feels a lot clearer, nice one.